pkg/mimirtool/client/client.go

```diff
@@ -51,6 +52,7 @@ type MimirClient struct {
 	Client    http.Client
 	apiPath   string
```

Comment:
Since `apiPath` is actually specific to the ruler, can you rename this member to `rulerApiPath`? Otherwise it's a bit confusing to have both `apiPath` and `apiPrefix`.